src/ORM/TableRegistry.php

```diff
@@ -184,7 +184,7 @@ public static function get($alias, array $options = [])
         }
 
         if (isset(static::$_config[$alias])) {
-            $options += static::$_config[$alias];
+            $options = array_merge($options, static::$_config[$alias]);
```

Comment:
If you plan on changing the order, you can still stick to + (if those are all string key based):
```php
$options = static::$_config[$alias] + $options;
```